frontend/lib/types/QualifiedType.cpp

```diff
@@ -69,7 +69,7 @@ void QualifiedType::mark(Context* context) const {
   context->markPointer(param_);
 }
 
-static const char* kindToString(QualifiedType::Kind kind) {
+const char* QualifiedType::kindToString(QualifiedType::Kind kind) {
```

Comment:
This seems to conflict with the addition of a static `kindToString` in `QualifiedType.h`. Seems to me like this should be static

Reply:
This implements the static `kindToString` method declared in `QualifiedType.h`, but you don't need to specify `static` on the implementation, just the declaration (as far as I know).

Reply:
Oop, I confused myself since the declaration wasn't present in the header before.